### DISCUSSION TITLE
Move to another code hosting platform?

### DISCUSSION THREAD

**Discussion by Suzie97** [nahuelwexd/vala-website#121] [General]
As sone you might know, after GitHub copilot more and more people talking about GitHub being anti-foss. 

More information here: https://sfconservancy.org/GiveUpGitHub/

Can this project afford to switch from GitHub?

**Comment by nyx-lyb3ra** [comment]
It was planned to move this repository to the Vala organization on GitHub, along with the other language-related projects.

I understand the motivations for seeking to migrate to platforms other than GitHub, but some community members had expressed a preference for GitHub because it increased the visibility of the language, and thus the number of potential contributors.

I would then solicit feedback from other members, such as @Prince781, for example.
